openapi/api-third-party.yaml

```diff
@@ -0,0 +1,42 @@
+swagger: '2.0'
+info:
+  title: Third Party Messages - Server implementation
+  version: '1'
+basePath: /
+schemes:
+  - https
+paths:
+  '/{fiscal_code}/messages/{id}':
```

Comment:
I'm not sure it's safe to use the fiscal_code in path. Maybe we have to move it in the header?

Reply:
@AleDore